code/modules/religion/rites.dm

```diff
@@ -0,0 +1,59 @@
+/datum/religion_rites
+	var/name = "religious rite"
+	var/desc = "immm gonna rooon"
+	var/ritual_length = (10 SECONDS) //total length it'll take
+	var/list/ritual_invocations //strings that are by default said evenly throughout the rite
+	var/favor_cost = 0
+	var/datum/religion_sect/owned_sect
```

Comment:
Why have this if the sect is a global var

Reply:
the simple fact that i am dumb